### PR TITLE
Renames ticker to SSticker and removes checks for non-existant modes.

### PR DESCRIPTION
**Pull request by DominikPanic**
:atom: for spookster

### CODE REVIEW

File: code/datums/mind.dm [hunk 1]
```diff
@@ -473,8 +473,8 @@ datum/mind
 	else
 		mind = new /datum/mind(key)
 		mind.original = src
-		if(ticker) ticker.minds += mind
-		else log_world("## DEBUG: mind_initialize(): No ticker ready yet! Please inform Carn")
+		if(SSticker) SSticker.minds += mind
+		else log_world("## DEBUG: mind_initialize(): No SSticker ready yet! Please inform Carn")
```
Comment (Menshin) [inline code/datums/mind.dm]: Carn has left the project since 5 years, may want to update the log.

File: code/modules/power/turbine.dm [hunk 2]
```diff
@@ -167,10 +167,6 @@
 		return
 	if (usr.is_mob_incapacitated(TRUE))
 		return
-	if (!(ishuman(usr) || ticker) && ticker.mode.name != "monkey")
```
Comment (Menshin) [inline code/modules/power/turbine.dm]: Won't removing the `ishuman(usr)` check allow every mob type to start the turbine ?

Comment (DominikPanic) [reply code/modules/power/turbine.dm]: Good catch I added some more checks.

File: code/datums/mind.dm (hunk 1 above)
Comment (DominikPanic) [reply code/datums/mind.dm]: Carn~~

File: code/modules/power/turbine.dm (hunk 2 above)
Comment (Ghommie) [reply code/modules/power/turbine.dm]: This check is an outdated version of `usr.IsAdvancedToolUser()` anyway, fyi.

File: code/game/sound.dm [hunk 3]
```diff
@@ -100,9 +100,9 @@
 	to_chat(src, S)
 
 /client/proc/playtitlemusic()
-	if(!ticker || !ticker.login_music)	return FALSE
+	if(!SSticker || !SSticker.login_music)	return FALSE
```
Comment (Rohesie) [inline code/game/sound.dm]:
```suggestion
	if(!SSticker?.login_music)	return FALSE
```

File: code/game/turfs/space.dm [hunk 4]
```diff
@@ -79,11 +79,10 @@
 
 	inertial_drift(A)
 
-	if(ticker && ticker.mode)
+	if(SSticker && SSticker.mode)
```
Comment (Rohesie) [inline code/game/turfs/space.dm]:
```suggestion
	if(SSticker?.mode)
```

File: code/modules/admin/debug_verbs.dm [hunk 5]
```diff
@@ -199,7 +199,7 @@ GLOBAL_PROTECT(AdminProcCallSpamPrevention)
 		else
 			return
 
-	if(!X || !istype(X) || X.gc_destroyed || !ticker || X.hivenumber != hivenumber_status)
+	if(!X || !istype(X) || X.gc_destroyed || !SSticker || X.hivenumber != hivenumber_status)
```
Comment (Rohesie) [inline code/modules/admin/debug_verbs.dm]:
```suggestion
	if(!istype(X) || X.gc_destroyed || !SSticker || X.hivenumber != hivenumber_status)
```

File: code/modules/admin/topic.dm [hunk 6]
```diff
@@ -1194,7 +1194,7 @@
 		if(!check_rights(R_SERVER))
 			return
 
-		if(ticker && ticker.mode)
+		if(SSticker && SSticker.mode)
```
Comment (Rohesie) [inline code/modules/admin/topic.dm]:
```suggestion
		if(SSticker?.mode)
```

File: code/modules/cm_aliens/XenoStructures.dm [hunk 7]
```diff
@@ -748,7 +748,7 @@ TUNNEL
 		return
 
 	//Prevents using tunnels by the queen to bypass the fog.
-	if(ticker && ticker.mode && ticker.mode.flags_round_type & MODE_FOG_ACTIVATED)
+	if(SSticker && SSticker.mode && SSticker.mode.flags_round_type & MODE_FOG_ACTIVATED)
```
Comment (Rohesie) [inline code/modules/cm_aliens/XenoStructures.dm]:
```suggestion
	if(SSticker?.mode && SSticker.mode.flags_round_type & MODE_FOG_ACTIVATED)
```

File: code/modules/cm_preds/yaut_items.dm [hunk 8]
```diff
@@ -813,11 +813,11 @@
 		l_color = "#FFFF0C" //Yeller
 		SetLuminosity(4)
 		spawn(3000)
-			if(ticker && istype(ticker.mode,/datum/game_mode/huntergames)) loop_firetick()
+			if(SSticker && istype(SSticker.mode,/datum/game_mode/huntergames)) loop_firetick()
```
Comment (Rohesie) [inline code/modules/cm_preds/yaut_items.dm]:
```suggestion
			if(istype(SSticker?.mode,/datum/game_mode/huntergames)) loop_firetick()
```

File: code/modules/mob/dead/observer/observer.dm [hunk 9]
```diff
@@ -669,11 +669,11 @@
 	set name = "Hunter Games Vote"
 	set desc = "If it's on Hunter Games gamemode, vote on who gets a supply drop!"
 
-	if(!ticker || ticker.current_state < GAME_STATE_PLAYING || !ticker.mode)
+	if(!SSticker || SSticker.current_state < GAME_STATE_PLAYING || !SSticker.mode)
```
Comment (Rohesie) [inline code/modules/mob/dead/observer/observer.dm]:
```suggestion
	if(!SSticker?.mode || SSticker.current_state < GAME_STATE_PLAYING)
```

File: code/modules/mob/living/carbon/xenomorph/Castes/Predalien.dm [hunk 10]
```diff
@@ -87,9 +87,9 @@
 	set waitfor = 0
 	sleep(30)
 	if(!loc) return FALSE
-	if(ticker && ticker.mode && ticker.mode.predators.len)
+	if(SSticker && SSticker.mode && SSticker.mode.predators.len)
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/xenomorph/Castes/Predalien.dm]:
```suggestion
	if(length(SSticker?.mode?.predators))
```

File: code/modules/mob/living/carbon/xenomorph/Castes/Queen.dm [hunk 11]
```diff
@@ -402,8 +402,8 @@
 	var/queensWord = "<br><h2 class='alert'>The words of the queen reverberate in your head...</h2>"
 	queensWord += "<br><span class='alert'>[input]</span><br>"
 
-	if(ticker && ticker.mode)
-		for(var/datum/mind/L in ticker.mode.xenomorphs)
+	if(SSticker && SSticker.mode)
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/xenomorph/Castes/Queen.dm]:
```suggestion
	if(SSticker?.mode)
```

File: code/modules/mob/living/carbon/xenomorph/Evolution.dm [hunk 12]
```diff
@@ -112,7 +112,7 @@
 			to_chat(src, "<span class='warning'>There already is a living Queen.</span>")
 			return
 
-		if(hivenumber == XENO_HIVE_NORMAL && ticker && ticker.mode && hive.xeno_queen_timer)
+		if(hivenumber == XENO_HIVE_NORMAL && SSticker && SSticker.mode && hive.xeno_queen_timer)
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/xenomorph/Evolution.dm]:
```suggestion
		if(hivenumber == XENO_HIVE_NORMAL && SSticker?.mode && hive.xeno_queen_timer)
```

File: code/modules/mob/living/carbon/xenomorph/XenoProcs.dm [hunk 13]
```diff
@@ -7,8 +7,8 @@
 	if(!message)
 		return
 
-	if(ticker && ticker.mode && ticker.mode.xenomorphs.len) //Send to only xenos in our gamemode list. This is faster than scanning all mobs
-		for(var/datum/mind/L in ticker.mode.xenomorphs)
+	if(SSticker && SSticker.mode && SSticker.mode.xenomorphs.len) //Send to only xenos in our gamemode list. This is faster than scanning all mobs
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/xenomorph/XenoProcs.dm]:
```suggestion
	if(length(SSticker?.mode?.xenomorphs)) //Send to only xenos in our gamemode list. This is faster than scanning all mobs
```

File: code/modules/mob/living/carbon/xenomorph/login.dm [hunk 14]
```diff
@@ -1,3 +1,3 @@
 /mob/living/carbon/Xenomorph/Login()
 	..()
-	if(ticker && ticker.mode) ticker.mode.xenomorphs |= mind
+	if(SSticker && SSticker.mode) SSticker.mode.xenomorphs |= mind
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/xenomorph/login.dm]:
```suggestion
	if(SSticker?.mode) SSticker.mode.xenomorphs |= mind
```

File: code/modules/mob/new_player/new_player.dm [hunk 15]
```diff
@@ -149,24 +149,24 @@
 
 
 		if("late_join_xeno")
-			if(!ticker || ticker.current_state != GAME_STATE_PLAYING || !ticker.mode)
+			if(!SSticker || SSticker.current_state != GAME_STATE_PLAYING || !SSticker.mode)
```
Comment (Rohesie) [inline code/modules/mob/new_player/new_player.dm]:
```suggestion
			if(!SSticker?.mode || SSticker.current_state != GAME_STATE_PLAYING)
```

**Pull request merged**